test/test_helper.rb

```diff
@@ -20,5 +17,6 @@ def reset_db
   end
 end
 
+Dir.glob(File.expand_path("../support/models/*.rb", __FILE__)).each { |r| require r }
 ActiveRecord::Base.establish_connection adapter: "sqlite3", database: ":memory:"
-load File.dirname(__FILE__) + '/dummy/db/schema.rb'
+require_relative "support/schema.rb"
```

Comment:
Pull in just the models and the schema for the in memory DB. Not an entire app we never use.